airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_35_46_001__DefaultConnectionName.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.instance.configs.migrations;
+
+import static org.jooq.impl.DSL.asterisk;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.Record;
+import org.jooq.Result;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class V0_35_46_001__DefaultConnectionName extends BaseJavaMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(V0_35_46_001__DefaultConnectionName.class);
+
+  public static void defaultConnectionName(final DSLContext ctx) {
+    LOGGER.info("updating connection name column");
+    final Field<UUID> id = DSL.field("id", SQLDataType.UUID.nullable(false));
+    final Field<String> name = DSL.field("name", SQLDataType.VARCHAR(256).nullable(false));
+    List<Connection> connections = getConnections(ctx);
+
+    for (final Connection connection : connections) {
+      final Actor sourceActor = getActor(connection.getSourceId(), ctx);
+      final Actor destinationActor = getActor(connection.getDestinationId(), ctx);
+      final String connectionName = sourceActor.getName() + " -> " + destinationActor.getName();
+
+      ctx.update(DSL.table("connection"))
+          .set(name, connectionName)
+          .where(id.eq(connection.getConnectionId()))
+          .execute();
+    }
+  }
+
+  static <T> List<Connection> getConnections(final DSLContext ctx) {
+    final Field<String> name = DSL.field("name", SQLDataType.VARCHAR(36).nullable(false));
+    final Field<UUID> id = DSL.field("id", SQLDataType.UUID.nullable(false));
+    final Field<UUID> sourceId = DSL.field("source_id", SQLDataType.UUID.nullable(false));
+    final Field<UUID> destinationId = DSL.field("destination_id", SQLDataType.UUID.nullable(false));
+    final Result<Record> results = ctx.select(asterisk()).from(DSL.table("connection")).fetch();
+
+    return results.stream().map(record -> new Connection(
+        record.get(name),
+        record.get(id),
+        record.get(sourceId),
+        record.get(destinationId)))
+        .collect(Collectors.toList());
+  }
+
+  static <T> Actor getActor(final UUID actionDefinitionId, final DSLContext ctx) {
+    final Field<String> name = DSL.field("name", SQLDataType.VARCHAR(36).nullable(false));
+    final Field<UUID> actorDefinitionId = DSL.field("actor_definition_id", SQLDataType.UUID.nullable(false));
+    final Result<Record> results = ctx.select(asterisk()).from(DSL.table("actor")).where(actorDefinitionId.eq(actionDefinitionId)).fetch();
+
+    return results.stream().map(record -> new Actor(
+        record.get(name))).toList().get(0);
```

Comment:
```suggestion
    return results.stream()
        .map(record -> new Actor(record.get(name))).toList().get(0);
```
this is the conventional way of splitting this new line. prefer splitting it on the various stream operations as opposed to internal to them where possible

the other option is to get rid of the newline entirely in this case as it is not strictly needed. that's a style choice though. up to you.